### PR TITLE
Now Slider gets updated if the value prop changes

### PR DESCRIPTION
**Pull request by anarthal**
**What kind of change does this PR introduce?**

Bugfix: https://github.com/react-native-elements/react-native-elements/issues/2995

**Did you add tests for your changes?**

No, just ran the existing ones.

**If relevant, did you update the documentation?**

Not required.

**Summary**

Attempts to fix https://github.com/react-native-elements/react-native-elements/issues/2995

**Does this PR introduce a breaking change?**

No.

**Other information**

### CODE REVIEW

**Comment by vincentjames501** [comment]
This one is a pretty big issue as we can't use this component for things like audio/video playback at the moment. Would love to see this one merged!

File: src/slider/Slider.tsx [hunk 1]
```diff
@@ -180,6 +177,7 @@ const Slider: RneFunctionComponent<SliderProps> = (props) => {
     } else {
       didMountRef.current = true;
     }
+    prevPropValue.current = props.value || 0;
```
Comment (arpitBhalla) [inline src/slider/Slider.tsx:180]: I am not sure this will make any difference

**Review by arpitBhalla — CHANGES_REQUESTED** [review]

File: src/slider/Slider.tsx (hunk 1 above)
Comment (khushal87) [reply src/slider/Slider.tsx:180]: Same! @arpitBhalla

**Comment by flyingcircle** [comment]
Closing. Fixed in #3023

**Pull request closed without merging**